inc/functions.php

```diff
@@ -346,7 +347,7 @@ function get_default_content_id( $content_id ): int {
 function get_language_by_http_code( $http_code ) {
 
 	return resolve( 'multilingualpress.languages', Languages::class )
-		->get_language_by_http_code( (string) $http_code );
+		->get_language_by( LanguagesTable::COLUMN_HTTP_CODE, (string) $http_code );
```

Comment:
This change should have gone into a dedicated PR branched off `master`. Can you do this and revert here, please?
Also, I'm wondering if we should actually wrap the whole new functionality, instead of only allowing to query by HTTP code... The other function `get_language_field_by_http_code()`, can stay as it is, in my opinion.

Reply:
Done: https://github.com/inpsyde/MultilingualPress/pull/293